packages/app-accounts/src/Creator.tsx

```diff
@@ -230,17 +232,24 @@ class Creator extends React.PureComponent<Props, State> {
 
   renderModalContent () {
     const { t } = this.props;
+    const { address } = this.state;
 
     return [
       <Modal.Header key='header'>
         {t('seedWarning.header', {
-          defaultValue: 'Warning'
+          defaultValue: 'Important notice!'
         })}
       </Modal.Header>,
       <Modal.Content key='content'>
+        <p>
         {t('seedWarning.content', {
-          defaultValue: 'Before you continue, make sure you have properly backed up your seed in a safe place as it is needed to restore your account.'
+          defaultValue: 'We will provide you with a generated backup file after your account was created. Please make sure to save this file in a secure location as it is the only way to restore your account. As long as you have access to your account you can always redownload this file later.'
```

Comment:
grammar nitpick - ...after your account ~~was~~ _is_ created...

Reply:
That's totally fine! As a non-native speaker I'm happy get corrected on things like that